src/rebar_prv_compile.erl

```diff
@@ -217,6 +217,9 @@ copy(OldAppDir, AppDir, Dir) ->
 
 %% TODO: use ec_file:copy/2 to do this, it preserves timestamps and
 %% may prevent recompilation of files in extra dirs
+copy(Source, Source) ->
+    %% someone specified a src dir inside _build. weird but ok
+    ok;
```

Comment:
it's useful for providers and plugins that want to stage some files to _build precompilation. without the source/dest check cp_r fails and aborts messily